app/jobs/import_url_job.rb

```diff
@@ -51,8 +51,12 @@ def perform(file_set, operation, headers = {})
     # Range: header requesting only the first byte. The successful response status
     # code is 206 instead of 200, but that is enough to satisfy the #success? method.
     # @param uri [URI] the uri of the file to be downloaded
-    def can_retrieve?(uri)
-      HTTParty.get(uri, headers: { Range: 'bytes=0-0' }).success?
+    def can_retrieve?(uri, headers = {})
```

Comment:
Why not just use `BrowseEverything::Retriever.can_retrieve?` as per #3167 / 106af06e510?

Reply:
I've started a new branch using that, but `BrowseEverything::Retriever.can_retrieve?` also needs to use any auth. headers in the request, please see https://github.com/samvera/hyrax/compare/master...jrgriffiniii:browse-everything-1.0.0-update?expand=1 and https://github.com/samvera/browse-everything/compare/master...jrgriffiniii:can-retrieve-headers?expand=1